index.html

```diff
@@ -655,7 +657,41 @@ <h4>LET'S CONNECT</h4>
 
          </div>
 
-     </section>
+     </div>
+
+     <div class="contacto">
+
+         <div class="flex-column fondo">
+
+             <h2 class="margen">Contact me</h2>
+
+             <p class="margen">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it!</p>
```

Comment:
- [ ] Great job @maximoortelli and @ALPANNA-GH You have done a good job so far. There are just a few changes I will suggest you include and you'll be on your way to the next milestone. 
- [ ] I noticed that there is a bigger spacing between the title `Contact me` and the paragraph text kindly close it up so it looks similar to the Figma Design :heavy_check_mark: 
- [ ] Also, please ensure that there is similar spacing around these elements as well as seen in the Figma design :heavy_check_mark: 

| Your Design | The Figma Design |
|--------|--------|
| ![image](https://github.com/maximoortelli/Portfolio-setup-and-mobile--first/assets/102699958/41dfefcf-4176-4245-bd4b-a23baaa106c4) | ![image](https://github.com/maximoortelli/Portfolio-setup-and-mobile--first/assets/102699958/ba3d24e9-1ffe-465a-bd5c-2198cf4ab455) |

Reply:
Done, thank you NahnahAJ 👍😃